model/push_notification.go

```diff
@@ -10,8 +10,10 @@ import (
 )
 
 const (
-	PUSH_NOTIFY_APPLE   = "apple"
-	PUSH_NOTIFY_ANDROID = "android"
+	PUSH_NOTIFY_APPLE_V1   = "apple"
+	PUSH_NOTIFY_ANDROID_V1 = "android"
+	PUSH_NOTIFY_APPLE_V2   = "apple2"
+	PUSH_NOTIFY_ANDROID_V2 = "android2"
```

Comment:
It needs different monikers since the push certs will be different for the different apps.  0/5 on what we call them.